MH4U Dex/CoreDataController.m

```diff
@@ -68,8 +69,8 @@ - (void)tryLoadSequence
 //        NSLog(@"Quest Data loaded.");
 //        [self loadQuestDropDataWithContext:backgroundContext];
 //        NSLog(@"Quest Drop Data loaded.");
-        [self attemptSaveContext:backgroundContext];
-        NSLog(@"Core Data Context Saved.");
+//        [self attemptSaveContext:backgroundContext];
+//        NSLog(@"Core Data Context Saved.");
```

Comment:
I need to leave that in because I still use it whenever I need to change the prebuilt store.